config/default/forms/app/pnc_danger_sign_follow_up_mother.xml

```diff
@@ -181,10 +181,10 @@
             <refer_patient_note_1 tag="hidden"/>
             <refer_patient_note_2 tag="hidden"/>
             <custom_translations tag="hidden">
-              <custom_woman_label_translator/>
-              <custom_woman_label/>
-              <custom_woman_start_label_translator/>
-              <custom_woman_start_label/>
+              <custom_woman_label_translator tag="hidden"/>
+              <custom_woman_label tag="hidden"/>
+              <custom_woman_start_label_translator tag="hidden"/>
+              <custom_woman_start_label tag="hidden"/>
```

Comment:
I don't think you need to add these individually since hiding the group hides all inner elements from the reports view too.